x-pack/filebeat/input/httpjson/internal/v2/value_tpl.go

```diff
@@ -236,3 +242,26 @@ func mul(a, b int64) int64 {
 func div(a, b int64) int64 {
 	return a / b
 }
+
+func hmac_string(hmac_type string, hmac_key string, values ...string) string {
```

Comment:
```suggestion
func hmacString(hmacType string, hmacKey string, values ...string) string {
```

Just for consistency with the rest.